### ISSUE TITLE
The alternative languages don't fit into the graphical elements

### ISSUE THREAD

**Issue by JakubGIT** [nhn/tui.editor#1634]
## Describe the bug
when choosing 'de' as language, the UI buttons size is not large enough to take the whole names

## To Reproduce
```TypeScript
import '@toast-ui/editor/dist/i18n/de-de';

export const MyEditor = (reference: React.LegacyRef<Editor>, content: string): JSX.Element => (
    <Editor
        ref={reference}
        height="600px"
        initialEditType="markdown"
        initialValue={content}
        language="de"
        previewStyle="tab"
        useCommandShortcut
    />
);
```
## Expected behavior
Flexible element size

## Screenshots
![image](https://user-images.githubusercontent.com/49392205/124428956-8f488380-dd6d-11eb-8989-0fadbd6e3671.png)
![image](https://user-images.githubusercontent.com/49392205/124429022-a2f3ea00-dd6d-11eb-870a-37705666aee1.png)

## Desktop (please complete the following information):
 - OS: Windows 10
 - Browser chrome
 - Version 91.0.4472.124 (Official Build) (32-bit)

**Comment by js87zz** [comment]
@JakubGIT 
I will fix in `v3.0.2`. Thanks!

**Comment by js87zz** [comment]
@JakubGIT 
The bug has been fixed in `v3.0.2`. Thanks!

**Comment by JakubGIT** [comment]
thank you very much!
